README.md

```diff
@@ -1,2 +1,29 @@
+<!-- Logo -->
+<p align="center">
+  <img height="128" width="128" src="https://github.com/cerner/terra-docker/raw/master/terra.png">
+</p>
+
+<!-- Name -->
+<h1 align="center">
+  Terra Toolkit
+</h1>
+
+[![License](https://badgen.net/github/license/cerner/terra-docker)](https://github.com/cerner/terra-docker/blob/master/LICENSE)
+
 # terra-docker
-A repo for docker files supporting terra CI and development.
+
+This is a repo for docker files supporting terra CI and development.
+
+This repo contains docker files to create the following images:
+
+## terra-node-base
```

Comment:
Can we mention node 10 alpine image is used?

Reply:
mentioned